vlasovsolver/cpu_1d_plm.hpp

```diff
@@ -38,8 +38,10 @@ t=(v-v_{i-0.5})/dv where v_{i-0.5} is the left face of a cell
 The factor 2.0 is in the polynom to ease integration, then integral is a[0]*t + a[1]*t**2
 */
 
-inline void compute_plm_coeff(const Vec * const values, uint k, Vec a[2]){
-  const Vec d_cv=slope_limiter(values[k - 1], values[k], values[k + 1]);
+inline void compute_plm_coeff(const Vec * const values, uint k, Vec a[2], const Realv threshold){
+   // scale values closer to 1 for more accurate slope limiter calculation
+  const Realv scale = 1./threshold;
+  const Vec d_cv=slope_limiter(values[k - 1]*scale, values[k]*scale, values[k + 1]*scale)*threshold;
```

Comment:
This is the point where it really doesn't make a lot of sense to call it "threshould", IMHO. Maybe just make the scale a constant, instead?

Reply:
It could be a constant yes, but if someone decides to run Vlasiator in unforeseen physical configurations, this could then cause errors to crop. The threshold value is inherently tied to the numerical scale of the VDF, and will always be set by the user, which is why I chose it.

If it's really seen as necessary, it can of course be made into an additional config file parameter with a default value of e.g. 1.e-15.